openpype/scripts/ocio_config_op.py

```diff
@@ -0,0 +1,52 @@
+import click
+import json
+from pathlib2 import Path
+import PyOpenColorIO as ocio
+
+
+@click.group()
+def config():
+    pass
```

Comment:
With this change there is no need to call `cli.add_command(config)`. I would also recommend to rename `cli` function to `main` (it's not requirement, but convenience).
```suggestion

@click.group()
def cli():
    pass


@cli.command()
def config():
    pass
```

Reply:
this is not working AttributeError: 'Command' object has no attribute 'command'